CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(BENCHMARK_DOWNLOAD_DEPENDENCIES "Allow the downloading and in-tree buildi
 # This option can be used to disable building and running unit tests which depend on gtest
 # in cases where it is not possible to build or find a valid version of gtest.
 option(BENCHMARK_ENABLE_GTEST_TESTS "Enable building the unit tests which depend on gtest" ON)
+option(BENCHMARK_USE_BUNDLED_GTEST "Use bundled GoogleTest. If disabled, the packaged version will be used." ON)
```

Comment:
`packaged` is ambigious here.
Perhaps something along the lines of `, the find_package(GTest) will be used`

Reply:
Fixed.